workshops/discord_message_bot/README.md

```diff
@@ -0,0 +1,395 @@
+
+<img src="https://media4.giphy.com/media/pvl3qUsgblNOo/200.gif" width="380" alt="Zelda Gif">
+
+## Repl.it Setup
+
+We're going to use [Repl.it](https://repl.it/~) to host the bot. It is an online IDE that makes it easy to setup and run the bot!
+
+Create a new repl and use Node.js as the language.
+
+<img src="https://cloud-otu0relhe.vercel.app/0screenshot__1383_.png" width="600" alt="Node.js Repl">
+
+
+Let's start creating it's functions!
+
+<img src="https://media1.tenor.com/images/d17514e2c03ec6b0e67ba7f18439a011/tenor.gif" width="380" alt="Shawn Sheep Gif">
+
+First, create two files called index.js and msgs.json. 
+
+  - Your messages will be stored in msgs.json. 
+  
+  - JSON is a file format that allows you to store data as a JavaScript object (key/value pair). If you want to learn more, [check here](https://www.json.org/json-en.html). 
+  
+  - As with defining a normal JavaScript object, your .json file should contain brackets as shown below.
+  
+
+<img src="https://cloud-gh7l7h2q1.vercel.app/json_example.png" width="380" alt="Write Command Example">
+
+In your index.js file, you will start off with code that looks like this. It will be explained below.
+
+```js
+const Discord = require('discord.js')
+const client = new Discord.Client()
+const fs = require('fs')
+client.msgs = require('./msgs.json')
+prefix = '!'
+
+client.once('ready', () => {
+  console.log('Ready!')
+})
+
+client.on('message', (message) => {})
+
+client.login('token')
+```
+
+_Make sure you insert your bot token where 'token' is!_
+
+Let's start by explaining what the prefix is. It allows you to use an exclamation mark to call the bot! Some people have other prefixes such as a dash or pound symbol. It is up to you!
+
+```js
+prefix = '!'
+```
+
+You also most likely do not know what these two lines do.
+
+```js
+const fs = require('fs')
+client.msgs = require('./msgs.json')
+```
+
+The fs is a file system node module. You need it to write to files.
+
+The line after this says that the json file we created will have messages that will be written to it.
+
+```js
+client.once('ready', () => {
+  console.log('Ready!')
+})
+```
+
+This part just let's you know the bot is on when you run it.
+
+## Write Command
+
+<img src="https://i.pinimg.com/originals/ca/38/b8/ca38b8f7e096956f258d406a7820c7ba.gif" width="380" alt="Snoopy Gif">
+
+Now let's start with the write message command. This will allow you to add your own custom messages. The user will provide a key that the message will be saved to and the message itself.
+
+<img src="https://cloud-kb9kganvz.vercel.app/write_message.png" width="380" alt="Write Command Example">
+
+Within your client.on brackets add this if statement.
+
+```js
+client.on('message', (message) => {
+  if (message.content.startsWith(`${prefix}write `)) {
+  
+  }
+})
+```
+
+The message.content part of this just looks at the message that the user typed.
+
+What this does in this case is make sure your bot command starts with !write
+
```

Comment:
I can't really put into words why, but these four paragraphs read awkwardly to me.